README.md

```diff
@@ -90,25 +91,19 @@ $ make binary-cross
 `kompose` is a work in progress, we will see how far it takes us. We welcome any pull request to make it even better.
 If you find any issues, please [file it](https://github.com/kubernetes-incubator/kompose/issues).
 
-## Community, discussion, contribution, and support
-
-We follow the Kubernetes community principles.
-
-Learn how to engage with the Kubernetes community on the [community page](http://kubernetes.io/community/).
-
-You can reach the maintainers of this project at:
+## Community, Discussion, Contribution, and Support
 
-- [Slack](http://slack.kubernetes.io): #kompose
+As part of the Kubernetes ecosystem, we follow the Kubernetes community principles. More information can be found on the [community page](http://kubernetes.io/community/).
 
-Incubation of `kompose` into the Kubernetes project is shepherd in the [SIG-APPS](https://github.com/kubernetes/community).
+You can reach the maintainers of this project on [Slack](http://slack.kubernetes.io) in channel #kompose
 
-[@ericchiang](https://github.com/ericchiang) is acting as Champion for [incubation](https://github.com/kubernetes/community/blob/master/incubator.md)
+`kompose` is being incubated into the Kubernetes community via [SIG-APPS](https://github.com/kubernetes/community/tree/master/sig-apps) on [kubernetes/community](https://github.com/kubernetes/community).
 
-We do a bi-weekly community meeting. Here is the link to [agenda doc](https://docs.google.com/document/d/1I5I21Cp_JZ9Az5MgMcu6Hl7m8WQ1Eqk_WeQLHenNom0/edit?usp=sharing).
+[@ericchiang](https://github.com/ericchiang) is acting champion for [incubation](https://github.com/kubernetes/community/blob/master/incubator.md).
 
-Meeting link: [https://bluejeans.com/404059616](https://bluejeans.com/404059616)
+We do a biweekly community meeting which is [open to the public](https://bluejeans.com/404059616). Each week we outline what we have talked about in an [agenda doc](https://docs.google.com/document/d/1I5I21Cp_JZ9Az5MgMcu6Hl7m8WQ1Eqk_WeQLHenNom0/edit?usp=sharing).
```

Comment:
May change each week, so it's in the Google doc. I'll update this PR with that change.